### DISCUSSION TITLE
Release 2.3.4

### DISCUSSION THREAD

**Discussion by androidacy-user** [Androidacy/MagiskModuleManager#82] [General]
[Read about and get downloads for this release here](https://www.androidacy.com/androidacy-module-manager-v2-3-4-updates-to-the-best-module-manager-app/?utm_source=github&utm_medium=web)

_Not a new release, but release notes were not published to Github releases due to an error in our automation. We regret this oversight._

<hr /><em>This discussion was created from the release <a href='https://github.com/Androidacy/MagiskModuleManager/releases/tag/v2.3.4'>Release 2.3.4</a>.</em>

**Comment by peggy-48** [comment]
As Rijk000 said, please remove the trackers and websites there. I don't know but i know a hosting service that ACTUALLY IS REALLY FREE called InfinityFree, here's the link, i even have a website for me with those great guys: [https://www.infinityfree.com](url), so why the hell do you use "telemetry" (_maybe i should just shoot myself down with a gun instead of saying that but..._), if you use the FOSS logic provided by Rijk, well the users might be more inclined to donate, or just stick to InfinityFree but please just stop putting spyware and ads on your website.

**Reply by androidacy-user** [reply]
> As Rijk000 said, please remove the trackers and websites there. I don't know but i know a hosting service that ACTUALLY IS REALLY FREE called InfinityFree, here's the link, i even have a website for me with those great guys: [https://www.infinityfree.com](url), so why the hell do you use "telemetry" (_maybe i should just shoot myself down with a gun instead of saying that but..._), if you use the FOSS logic provided by Rijk, well the users might be more inclined to donate, or just stick to InfinityFree but please just stop putting spyware and ads on your website.

Are you going to start paying our staff too? Paying for the software we use?

And above all - do you _really_ think we haven't investigated services like that?

Last week alone, **JUST OUR API** saw ~1.5M requests, generating over a terabyte of total traffic, spread across the globe. That _does not_ include our website or bitwarden server (combined, those saw around ~5M requests and around 2.5TB). A lot of this content can't be served via our cdn provider for various reasons, and has to be served by our local data center. We have ~3-10 servers per region, which varies depending on traffic load and is designed to scale on demand. We also require various staff to maintain those services, provide support for our users, and maintain our website and blog.

And - above all - **people don't donate unless they get something in return, which really defeats the purpose**. It's not just us. Most developers who rely on donations alone see very, very little reward for their work. Even John Wu - y'know the guy who created a tool basically every rooted android user is using right now - doesn't see that many donations, based on the figures we were able to find. Stop suggesting that, it's frankly insulting to any developers, not just us.

And telemetry/analytics allows us insights into who is using our services where and what's going wrong, so that we can better serve our users and focus our energy and strategy.

Good day.

**Comment by VA1DER** [comment]
> As Rijk000 said, please remove the trackers and websites there.

This is less than unlikely to happen. There is more to this than just privateering.  

The vacuum created by topjohnwu's removal of the module repository (and magiskhide) features at the behest of his new masters is what makes this sort of behaviour possible.  Just as the original vacuum was created by Google, it also serves Google's purpose to gradually discredit the root module scene.  If it becomes a den of thieves, rather than perceived as a valid open source ecosystem, then Google can at some point take definitive steps to end it and give themselves the control that Apple has.

As it stands, they can't do that quite yet.  So they move behind the scenes to increase the noise and discredit the scene.  There is more to [@Fox2Code](https://github.com/Fox2Code)'s departure.  I highly suspect there was an exchange of money and an NDA involved.  I would also not be surprised to find that Google is involved somewhere.

Regardless, it's still unlikely once the sleazy size gets it mitts on it that they will just back off and turn it back into an open and reputable system again.

We should get together and see about a fork and moving the repository.  A code of conduct for the repository would be good too - only open source modules would be a great start, with negative labels as modules incorporate what (to the open source community) are anti-features. We could make it into the f-droid of magisk modules.

@HuskyDG is doing some great work on versions of Magisk that re-incorporate magiskhide and better zygote.  If we do fork this and get a good repo going, I suspect it wouldn't be a stretch to getting it incorporated by default into his builds.  We could put humpty back together again.
